package.json

```diff
@@ -12,7 +12,7 @@
   "dependencies": {
     "@docusaurus/core": "2.1.0",
     "@docusaurus/preset-classic": "2.1.0",
-    "@docusaurus/theme-search-algolia": "2.1.0",
+    "@docusaurus/theme-search-algolia": "^2.2.0",
```

Comment:
Can we leave the version update to another PR?

Reply:
@yuchaoran2011 Sure!

Reply:
Looks like this change hasn't been reverted back

Reply:
@yuchaoran2011 Nailed it. Thanks!